scripts/qa_crowbarsetup.sh

```diff
@@ -799,8 +799,10 @@ BROADCAST='$net.255'
 EOF
     ifdown br0
     rm -f /etc/sysconfig/network/ifcfg-br0
-    grep -q "^default" /etc/sysconfig/network/routes || \
```

Comment:
I wonder why we needed this grep at all - added in 386064c4bbada0d4c8170de5f9e9ece546fcb178 by @jdsn